src/helpers.js

```diff
@@ -32,40 +32,42 @@ const CONSOLE_API_KEYS = {
  * @returns {any} Returns a config object or null
  */
 async function getCommerceAdminConfig() {
-	const configFile = Config.get('aio-cli-plugin-commerce-admin');
+	const configFile = Config.get('mesh-api-manager.configPath');
 
 	if (!configFile) {
 		return {
-			baseUrl: 'https://graph.adobe.io/api-admin',
+			baseUrl: 'https://developers.adobe.io/console',
 			accessToken: (await getLibConsoleCLI()).accessToken,
-			apiKey: 'graphql-onboarding-io',
+			apiKey: 'adobe-api-manager-sms-stage',
```

Comment:
We need to get an API key for prod. `adobeio_onboarding` is probably not the right key to use in prod.